packages/gvim.rb

```diff
@@ -3,81 +3,50 @@
 class Gvim < Package
   description 'gVim is a highly configurable text editor built to make creating and changing any kind of text very efficient.'
   homepage 'http://www.vim.org/'
-  version '8.1-1'
+  version '8.1-2'
   source_url 'ftp://ftp.vim.org/pub/vim/unix/vim-8.1.tar.bz2'
   source_sha256 '8b69fbd01c877dd8ecbbeca1dc66e5e927228d631ac4c2174b9307eb5c827c86'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gvim-8.1-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gvim-8.1-1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gvim-8.1-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gvim-8.1-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'fdda85b0be941d9043acf35f166b7c95457ab96d04c8a3c82b96c7bc6314f99e',
-     armv7l: 'fdda85b0be941d9043acf35f166b7c95457ab96d04c8a3c82b96c7bc6314f99e',
-       i686: 'dcf0668bbabdd5623ed6481c3e1503a52df398771fe3c9fa64a1f58d1cb0ed95',
-     x86_64: 'd8fb8c1ee73295d0e7646235c012f1785c38a67c46db18846faf33b89384cdfc',
   })
 
   depends_on 'vim'
   depends_on 'gtk2'
   depends_on 'sommelier'
+  depends_on 'perl' => :build
+  depends_on 'python27' => :build
+  depends_on 'python3' => :build
+  depends_on 'ruby' => :build
 
   def self.build
     system "./configure \
             --prefix=#{CREW_PREFIX} \
-            --disable-nls \
-            --enable-gui \
+            --localstatedir=#{CREW_PREFIX}/var/lib/vim \
+            --with-features=huge \
+            --with-compiledby='Chromebrew' \
+            --with-tlib=ncurses \
```

Comment:
If possible, can we use `ncursesw`?

Reply:
It was `ncurses` previously so I didn't mess with it, I'll try building it with `w` and see what happens.